variables.tf

```diff
@@ -181,9 +181,9 @@ variable "authentication_cognito_user_pool_domain" {
 }
 
 variable "authentication_cognito_scope" {
-  type        = list(string)
+  type        = string
   description = "Cognito scope"
```

Comment:
@jhosteny can you provide some context here (and in the other var) in the description on how to properly encode the string for whatever Idps you know? I think that will help folks who upgrade to this new version.

Reply:
Hi @Gowiem, done. Note that I elected to pass this through as a string, rather than assemble it from the component values with an explicit join of a space character. Previously, we joined the values with an _encoded_ space character, which was incorrect since the ALB already does the encoding for you. We are not entirely certain why we did not catch this before.